bundles/SystemInfoBundle/public/js/startup.js

```diff
@@ -0,0 +1,99 @@
+        });
+
+        if (filteredMenu.length > 0) {
+            const systemInfoMenu = filteredMenu.shift();
+            systemInfoMenuItems.map(function(item) {
+                systemInfoMenu.menu.items.push(item);
+            });
+        } else {
+            menu.extras.items.push({
+                text: t("system_infos_and_tools"),
+                iconCls: "pimcore_nav_icon_info",
+                hideOnClick: false,
+                itemId: 'pimcore_menu_extras_system_info',
+                menu: {
+                    cls: "pimcore_navigation_flyout",
+                    shadow: false,
+                    items: systemInfoMenuItems
+                }
+            })
+        }
+    },
+
+    getSystemInfoMenu: function () {
+        const items = [];
+
+        if (this.user.admin && this.perspectiveConfig.inToolbar('extras.systemtools')) {
+
+            if (this.perspectiveConfig.inToolbar('extras.systemtools.phpinfo')) {
+                items.push(
+                    {
+                        text: t('php_info'),
+                        iconCls: 'pimcore_nav_icon_php',
+                        itemId: 'pimcore_menu_extras_system_info_php_info',
+                        handler: this.showPhpInfo
+                    }
+                );
+            }
+
+            if (this.perspectiveConfig.inToolbar('extras.systemtools.opcache')) {
+                items.push(
+                    {
+                        text: t('php_opcache_status'),
+                        iconCls: 'pimcore_nav_icon_reports',
+                        itemId: 'pimcore_menu_extras_system_info_php_opcache_status',
+                        handler: this.showOpcacheStatus
+                    }
+                );
+            }
+
+            if (this.perspectiveConfig.inToolbar('extras.systemtools.requirements')) {
+                items.push(
+                    {
+                        text: t('system_requirements_check'),
+                        iconCls: 'pimcore_nav_icon_systemrequirements',
+                        itemId: 'pimcore_menu_extras_system_info_system_requirements_check',
+                        handler: this.showSystemRequirementsCheck
+                    }
+                );
+            }
+        }
+
+        return items;
+    },
+
+    showPhpInfo: function () {
+        pimcore.helpers.openGenericIframeWindow("phpinfo", Routing.generate('pimcore_admin_misc_phpinfo'), "pimcore_icon_php", "PHP Info");
+    },
+
+    showOpcacheStatus: function () {
+        pimcore.helpers.openGenericIframeWindow("opcachestatus", Routing.generate('pimcore_admin_external_opcache_index'), "pimcore_icon_reports", "PHP OPcache Status");
+    },
+
+    showSystemRequirementsCheck: function () {
+        pimcore.helpers.openGenericIframeWindow("systemrequirementscheck", Routing.generate('pimcore_admin_install_check'), "pimcore_icon_systemrequirements", "System-Requirements Check");
```

Comment:
The 3 routing.generate prefix need to be changed eg.
```suggestion
        pimcore.helpers.openGenericIframeWindow("phpinfo", Routing.generate('pimcore_bundle_systeminfo_phpinfo'), "pimcore_icon_php", "PHP Info");
```